korman/nodes/node_python.py

```diff
@@ -751,10 +751,10 @@ def draw_buttons(self, context, layout):
         layout.prop(self, "target_object", text=self.attribute_name)
 
     def get_key(self, exporter, so):
-        attrib = self.to_socket
-        if attrib is None:
+        pynode = self.to_socket
+        if pynode is None:
             self.raise_error("must be connected to a Python File node!")
-        attrib = attrib.attribute_type
+        attrib = pynode.attribute_type
```

Comment:
I think your new variable name is a bit misleading.. We're getting the attribute's socket on the python file node, not the python file node itself. `attrib_socket` would probably be more appropriate.